### ISSUE TITLE
Typo at page 69 of the PDF version

### ISSUE THREAD

**Issue by hdcpereira** [JuliaDataScience/JuliaDataScience#105]
As in the web version, where its written: 

```
DataFrame(σ = ["a", "a", "a"], δ = [π, π/2, π/3])
```
The PDF version reads:
```
DataFrameσ( = ["a", "a", "a"], δ = π[, π/2, π/3])
```
page 69 Chapter 4 DataFrames.jl

**Comment by hdcpereira** [comment]
actually, as I go through the book's PDF version I'm finding a lot of typos. I prefer the PDF version but it seems like the web version is far more accurate. For the person assigned for this issue, do you want me to keep updating this topic with the typos I find as I read it or maybe not?

**Comment by rikhuijzer** [comment]
Oh wow. That's not a typo but a serious bug in either Pandoc, Books.jl or the JuliaMono font. I'll investigate it soon. Thanks for reporting the issue!

> do you want me to keep updating this topic with the typos I find as I read it or maybe not?

The PDF is automatically generated from the same source as the HTML, so I guess they are all caused by the same bug. Do the typos look like they could have the same source, or do you see wildly different typos? If you could report a few different types, then I can fix it and double check whether all typos have been fixed.

**Comment by hdcpereira** [comment]
> Do the typos look like they could have the same source, or do you see wildly different typos?

Yes, it seems like there's a problem when generating code cells. At page 73 when presenting the results of a CSV.read, the output looks messy for both functions. The same happens at page 84, 85, 86, 87 and 97. At glance I don't see any other mistakes in the code sections I read. I can't say for the whole book, but these are the only typos I found.

**Comment by storopoli** [comment]
Thanks @hdcpereira, we'll try to get it sorted out. We are in the middle of the final push of the book and discovering that our PDF is seriously broken is bad news.

**Comment by rikhuijzer** [comment]
**TLDR**: Should be fixed now. Let me know if you spot more wrongly located symbols.

This is fixed, and I've set up syntax highlighting too.

It was a particularly nasty issue in my opinion. Basically, the cause was that the `listings` package in LaTeX didn't recognize some unicode symbols and, therefore, decided to **move them to another place** even ignoring newlines. I didn't check the full document, maybe some symbols are still missing. If that is the case **please let me know** and I'll make some more PRs like the one at https://github.com/mossr/julia-mono-listings/pull/7.

Regarding the syntax highlighting, the colors currently differ from the ones at the website. To fix this, we have to update 

https://github.com/mossr/julia-mono-listings/blob/a5bd1d8aa5dc918ddc56f95439accc3ddfb50c91/julia_listings.tex#L6-L23

which is something for later.
